lib/uber/api/products.rb

```diff
@@ -9,6 +9,14 @@ def products(*args)
         arguments = Uber::Arguments.new(args)
         perform_with_objects(:get, "/v1/products", arguments.options, Product)
       end
+
```

Comment:
@chrisenytc can you resubmit the pull request by replacing tab with "2 space", thank you!